packages/wrangler/src/pages/build.ts

```diff
@@ -235,6 +235,7 @@ export const Handler = async (args: PagesBuildArgs) => {
 					buildOutputDirectory,
 					nodejsCompat,
 					defineNavigatorUserAgent,
+					sourceMaps: config?.upload_source_maps ?? false,
```

Comment:
There are various other bundling functions in this file that should probably also take this argument (e.g. `buildRawWorker()`)